NachoSpigot-API/src/main/java/com/google/common/util/concurrent/MoreExecutors.java

```diff
@@ -436,18 +436,26 @@ public static ListeningExecutorService newDirectExecutorService() {
      * Executor#execute execute}, as in {@code ThreadPoolExecutor.CallerRunsPolicy}.
      *
      * <p>This executor is appropriate for tasks that are lightweight and not deeply chained.
-     * Inappropriate {@code directExecutor} usage can cause problems, and these problems can be
+     * Inappropriate {@code directExecutor} usage can cause problems, and these problems can bef
```

Comment:
@Elierrr intended?

Reply:
That was in Guava javadoc, though it can be fixed here.
```suggestion
     * Inappropriate {@code directExecutor} usage can cause problems, and these problems can be
```